cooltools/lib/common.py

```diff
@@ -512,7 +507,10 @@ def align_track_with_cooler(
 
 def pool_decorator(func):
     """
-    A decorator function that enables multiprocessing for a given function.
+    A decorator function that enables multiprocessing for a given function.  
+    Function must have nproc and map_functor as key word arguments.  
```

Comment:
The wrapped function doesn't actually need `nproc`, does it? The wrapper takes it in and uses it to initialize a functor.